app/assets/javascripts/models/beta/attachmentUrl.js

```diff
@@ -0,0 +1,16 @@
+export const cloudnaryUrl = () =>
+  process.env.CLOUDINARY_URL;
+
+export const cloudName = () =>
+  cloudnaryUrl().substring(cloudnaryUrl().indexOf('@') + 1);
+
+export const uploadPreset = () =>
+  process.env.CLOUDINARY_UPLOAD_PRESET;
+
+export const uploadUrl = () => {
+  return `https://api.cloudinary.com/v1_1/${cloudName()}/upload`
```

Comment:
Should this API version be hardcoded here? What about moving it to somewhere else?

Reply:
I think is fine to be here, is encapsulated in this `attachmentUrl` file :thinking: . But what you suggest @vitortalaia ?.